File: Java/Film.java
import java.util.Scanner;

// Membuat kelas Film
public class Film {
    // Atribut
    private int IdFilm;
    private String Judul;
    private String Genre;
    private int HargaTiket;
    private int Studio;

    // Konstruktor
    public Film(){ 
    }
    public Film(int IdFilm, String Judul, String Genre, int HargaTiket, int Studio){
        this.IdFilm = IdFilm;
        this.Judul = Judul;
        this.Genre = Genre;
        this.HargaTiket = HargaTiket;
        this.Studio = Studio;
    }

    // Setter
    public void setIdFilm(int IdFilm){
        this.IdFilm = IdFilm;
    }

    public void setJudul(String Judul){
        this.Judul = Judul;
    }
    
    public void setGenre(String Genre){
        this.Genre = Genre;
    }
    
    public void setHargaTiket(int HargaTiket){
        this.HargaTiket = HargaTiket;
    }
    
    public void setStudio(int Studio){
        this.Studio = Studio;
    }
    
    // Getter
    public int getIdFilm(){
        return IdFilm;
    }

    public String getJudul(){
        return Judul;
    }

    public String getGenre(){
        return Genre;
    }

    public int getHargaTiket(){
        return HargaTiket;
    }

    public int getStudio(){
        return Studio;
    }
}

File: PHP/Film.php
<?php

class Film {
    // Atribut yang dimiliki oleh object Film
    private int $IdFilm;
    private string $Judul;
    private string $Genre;
    private int $HargaTiket;
    private int $Studio;
    private string $Gambar;

    // Constructor untuk mengisi data awal object Film
    public function __construct(int $IdFilm, string $Judul, string $Genre, int $HargaTiket, int $Studio, string $Gambar){
        $this->IdFilm = $IdFilm;
        $this->Judul = $Judul;
        $this->Genre = $Genre;
        $this->HargaTiket = $HargaTiket;
        $this->Studio = $Studio;
        $this->Gambar = $Gambar;
    }

    // Setter digunakan untuk mengubah nilai atribut
    public function setIdFilm(int $IdFilm): void {
        $this->IdFilm = $IdFilm;
    }

    public function setJudul(string $Judul): void {
        $this->Judul = $Judul;
    }

    public function setGenre(string $Genre): void {
        $this->Genre = $Genre;
    }

    public function setHargaTiket(int $HargaTiket): void {
        $this->HargaTiket = $HargaTiket;
    }

    public function setStudio(int $Studio): void {
        $this->Studio = $Studio;
    }

    public function setGambar(string $Gambar): void {
        $this->Gambar = $Gambar;
    }

    // Getter digunakan untuk mengambil nilai atribut
    public function getIdFilm(): int {
        return $this->IdFilm;
    }

    public function getJudul(): string {
        return $this->Judul;
    }

    public function getGenre(): string {
        return $this->Genre;
    }

    public function getHargaTiket(): int {
        return $this->HargaTiket;
    }

    public function getStudio(): int {
        return $this->Studio;
    }

    public function getGambar(): string {
        return $this->Gambar;
    }
}